docs/reference/settings/ml-settings.asciidoc

```diff
@@ -119,7 +121,7 @@ concurrently be in the `opening` state on each node. Typically, jobs spend a
 small amount of time in this state before they move to `open` state. Jobs that
 must restore large models when they are opening spend more time in the `opening`
 state. When the {operator-feature} is enabled, this setting can be updated only 
-by operator users. Defaults to `2`.
+by operator users. The minimum value is `0`. Defaults to `2`.
```

Comment:
I think it's a bug that the minimum is 0 with no special meaning.  I opened #70064 to discuss what to do about this.

If we leave the default as 0 with no special meaning then I think it needs an extra sentence here to explicitly call out the effect of setting 0.